api/src/app.ts

```diff
@@ -32,6 +33,11 @@ app.use(favicon(path.join(app.get("public"), "favicon.ico")));
 // Host the public folder
 app.use("/", express.static(app.get("public")));
 
+//database health check
```

Comment:
Nit.: this is more of a service health check that depends on the db, than a db health check.